Cargo.toml

```diff
@@ -3,6 +3,18 @@ name = "oyster-attestation-verifier"
 version = "2.0.1"
 edition = "2021"
 
+[lib]
+name = "verifier"
+path = "src/lib.rs"
+
+[[bin]]
+name = "attestation-verifier"
```

Comment:
add oyster at start of the name of executable, to keep the same executable name as before.